_posts/2018-08-06-bashing-through-minikube.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  "Bashing through Minikube - 0 to 60 for Local Kubernetes Development"
+date:   2018-08-06
+tags: kubernetes minikube containers bash local development
+author: Josh Armitage
+image: img/kubernetes/minikube-logo.png
+---
+
+## Minikube You Say?
+When you begin on a Kubernetes journey, often one of the first questions is how do I run locally? That's where Minikube comes in, you get an easy to spin up single node Kubernetes environment where you can rapidly iterate. As with nearly all tools, there's some rough edges and gotchas to figure out, in the next 5 minutes I hope to save you hours of frustration.
```

Comment:
That's where Minikube comes in; you get an easy to spin up single node Kubernetes environment which allows you to rapidly iterate.

Reply:
potentially split the last sentence of the 1st para into two sentences.